esp-wifi/MIGRATING-0.9.md

```diff
@@ -60,3 +60,33 @@ You now need to have a global allocator provided by `esp-alloc` providing alloca
 
     // ...
 ```
+
+The size of the heap depends on what you are going to use esp-wifi for and if you are using the heap for your own allocations or not.
+
+e.g. when using `coex` you need around 92k, if not using `coex` going lower than 72k you will observe some failed allocations but it might still work - going even lower will make things fail
+
+### Using your own allocator
+
+You can also use your own allocator instead of using `esp-alloc`. To do that you need to opt-out of the default feature `esp-alloc` and provide two functions
```

Comment:
```suggestion
You can also use your own allocator instead of using `esp-alloc`. To do that you need to opt-out of the default feature `esp-alloc` and provide two functions:
```